vars/mavenIntegrationTest.groovy

```diff
@@ -29,7 +29,7 @@ def call(body) {
         echo "WARNING: Integration tests are current DISABLED for these pipelines!"
 
     } else {
-        sh "mvn org.apache.maven.plugins:maven-failsafe-plugin:2.18.1:integration-test ${kubeNS} -Dit.test=${config.itestPattern} -DfailIfNoTests=${config.failIfNoTests} org.apache.maven.plugins:maven-failsafe-plugin:2.18.1:verify"
```

Comment:
Could you please shed some light into the implication of removing the version?

Reply:
It will use the one user has defined in his build. E.g. latest one is `2.20.1` if I remember correctly and have numerous of fixes. Unless we exactly specify why `2.18.1` is needed I think we should safe users from such surprises.

Reply:
I have no idea (git blame doesn't explain) why the version was pinned in the first place. @bartoszmajsak thanks for the explanation thou.

Reply:
Welcome. We might think about making it configurable, but only if really necessary. Fixing version and having it potentially different from the user's choice add another layer of confusion in this lib :)